package.json

```diff
@@ -22,7 +23,11 @@
 		}
 	},
 	"dependencies": {
+		"fabric": "^5.2.4",
 		"handlebars": "^4.7.7",
 		"marked": "^4.1.0"
+	},
+	"alias": {
+		"buffer": false
```

Comment:
Prevent parcel from trying to polyfill this, since `buffer` is checked in fabric in order to know if it is running in an node environment (which is weird but whatever)